server/tests/tasks/test_github_repo_import.py

```
@@ -0,0 +1,140 @@
+        gen.delay.assert_not_called()  # already present → no metadata re-dispatch
+
+    def test_repo_missing_full_name_skipped(self):
+        repos = [{"id": 9, "default_branch": "main"}, {"full_name": "acme/ok", "id": 1}]
+        _, cur, gen, _ = _run(repos, existing=())
+        assert len(_insert_calls(cur)) == 1
+        assert {c.args for c in gen.delay.call_args_list} == {(_USER, "acme/ok")}
+
+    def test_suspended_installation_is_a_noop(self):
+        from utils.auth.github_app_token import GitHubAppInstallationSuspended
+
+        conn, cur, gen, fetch = _run(
```

Ruff (0.15.15)

[warning] Unpacked variable `conn` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)